docs/customization/custom-scripts.md

```diff
@@ -108,6 +108,10 @@ commit_default = False
 
 By default, a script can be scheduled for execution at a later time. Setting `scheduling_enabled` to False disables this ability: Only immediate execution will be possible. (This also disables the ability to set a recurring execution interval.)
 
+### `rq_queue_name`
```

Comment:
In #16927 I'm introducing a general purpose job API used by NetBox and made available for plugins. A minor background for this is abstraction to decouple plugins from using Redis directly, making future changes possible. Therefore I'd like to suggest not using parameter names related to Redis queues, i.e. just using something like `queue` instead of `rq_queue_name`.

Reply:
I'm good with whatever Netbox wants to use for name, be it `queue`, `worker_queue` or `job_queue`.  But I won't make any changes right now till the Netbox maintainers pick the language that is preferred for this.